### DISCUSSION TITLE
Bug with Telepathic Projection esoteric talent

### DISCUSSION THREAD

**Discussion by 1001100x02** [bcholmes/StarTrek2d20#164] [Ideas]
The Telepathic Projection esoteric talent only becomes an option when you choose the the Telepathy (Betazoid) talent.

It should also key off the Telepathy (Occampa) talent, especially given the write up for that makes development of the esoteric talents a really logical path of development

**Comment by Felderburg** [comment]
It is perhaps worth noting that the Kzinti Telepath talent is yet another variety of the talent (from The Animated Series Supplement, the talents are on my airtable spreadsheet).

It removes the non-optional reading of nearby surface thoughts, and adds a consequence to failing to read minds (which is, as I recall, consistent with what we saw in the episode).

**Comment by 1001100x02** [comment]
Yes, at this point it would seem logical to give telepathic powers a "telepath" tag, and let Teep. Projection key off that rather than code it to specific talents
